<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="19815" started-at="2017-05-17T13:56:26Z" finished-at="2017-05-17T13:56:46Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="19815" started-at="2017-05-17T13:56:26Z" finished-at="2017-05-17T13:56:46Z">
      <class name="com.cgi.code.testng.ExcelTestData">
        <test-method status="PASS" signature="BeforeClass()[pri:0, instance:com.cgi.code.testng.ExcelTestData@731a74c]" name="BeforeClass" is-config="true" duration-ms="4749" started-at="2017-05-17T10:56:26Z" finished-at="2017-05-17T10:56:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeClass -->
        <test-method status="FAIL" signature="DoTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.cgi.code.testng.ExcelTestData@731a74c]" name="DoTest" duration-ms="12654" started-at="2017-05-17T10:56:31Z" data-provider="name" finished-at="2017-05-17T10:56:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Regression]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Validate Mortgage Page]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[yes]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[ ]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected not same]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected not same
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.failSame(AssertJUnit.java:352)
	at org.testng.AssertJUnit.assertNotSame(AssertJUnit.java:310)
	at org.testng.AssertJUnit.assertNotSame(AssertJUnit.java:319)
	at com.cgi.code.testng.ExcelTestData.DoStep(ExcelTestData.java:331)
	at com.cgi.code.testng.ExcelTestData.DoTest(ExcelTestData.java:139)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DoTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.cgi.code.testng.ExcelTestData@731a74c]" name="TearDown" is-config="true" duration-ms="1933" started-at="2017-05-17T10:56:44Z" finished-at="2017-05-17T10:56:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.cgi.code.testng.ExcelTestData -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
